### ISSUE TITLE
[BUG] DaisyUI class "menu" is reported as ab error 'no-custom-class'

### ISSUE THREAD

**Issue by nader-eloshaiker** [francoismassart/eslint-plugin-tailwindcss#270]
**Describe the bug**
ESLint reports:
```
.../src/components/autocomplete/Autocomplete.tsx
  79:13  warning  Classname 'menu-compact' is not a Tailwind CSS class!  tailwindcss/no-custom-classname
```

**To Reproduce**
```
        <div className='menu menu-compact'>
         Error
       </div>

```
**Expected behavior**
Should support this classname

**Environment (please complete the following information):**
 - "eslint-plugin-tailwindcss": "^3.13.0"
 - "eslint": "^8.48.0"
 - "daisyui": "^3.1.6"
 - "tailwindcss": "^3.3.3"

**Additional context**
I see your version of DaisyUI in your dev dependencies, is at `"daisyui": "^2.6.4"`, could potentially be the cause.

**eslint config file or live demo**
No custom config for this plugin and appears to be working otherwise.
